sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/RateLimitingSampler.java

```diff
@@ -62,11 +64,19 @@ public SamplingResult shouldSample(
 
   @Override
   public String getDescription() {
-    return String.format("RateLimitingSampler{%.2f}", maxTracesPerSecond);
+    return String.format("RateLimitingSampler{%s}", decimalFormat(maxTracesPerSecond));
```

Comment:
#nit: now that we're not relying on any decimal formatting, let's switch to string concatenation, which is generally favored over `String.format` in this repository: `"RateLimitingSampler{" + decimalFormat(maxTracesPerSecond) + "}"`

Same advice applies in `TraceIdRatioBasedSampler`.